configuration/config.py.TEMPLATE

```diff
@@ -0,0 +1,47 @@
+import os
```

Comment:
Name this file `configuration/config.py.TEMPLATE` instead.  With the `.py` extension at the end of the name, editors and IDEs will be able to work with it much easier.

Also, I think the project already has a "default"/"dev"/"template" version of this file.

Reply:
Nobody is supposed to work with the TEMPLATE file.  They are supposed to copy it and work with the copy.  But I'll revisit this setup later.

Reply:
Right.  But as a developer writing or updating this file in the project, it would be very helpful for the editor to know what kind of file it is.